Libraries/CustomComponents/ListView/ListView.js

```diff
@@ -229,6 +223,14 @@ var ListView = React.createClass({
      */
     removeClippedSubviews: React.PropTypes.bool,
     /**
+     * Makes the sections headers sticky.  The sticky behavior means that it
+     * will scroll with the content at the top of the section until it reaches
+     * the top of the screen, at which point it will stick to the top until it
+     * is pushed off the screen by the next section header.
+     * @platform ios
+     */
+    stickySectionHeaders: React.PropTypes.bool,
```

Comment:
@Kureev an extra flag is needed if we implement the sticky section header on Android as they have the different behavior by default
There is a discussion here #9957 and @janicduplessis did the same thing as @matto1990 did, though that PR was reverted then